standalone_tags:
  [
    {id: 55, size: 0.058},
    {id: 56, size: 0.058}
  ]
tag_bundles:
  [
  ]
